angular/src/app/app.component.html

```diff
@@ -0,0 +1,185 @@
+<main class="max-w-3xl mt-5">
+  <h1 class="text-5xl pl-3">Web Chat</h1>
+  <div
+    class="bg-blue-100 border-l-4 border-blue-500 text-blue-800 p-2 my-4 rounded"
+  >
+    <p>
+      This page is created purely for demo purposes and does not contain all the
+      Web Chat APIs, options, or events. Please check the
+      <a
+        href="https://developers.helpshift.com/web-chat/"
+        class="text-blue-500 "
+        target="_blank"
+      >
+        Developer documentation
+      </a>
+      for the complete details.
+    </p>
+  </div>
+
+  <div class="w-full border-gray-300 mt-12 rounded-lg">
+    <h3
+      class="py-2 pl-3 text-lg rounded-t-lg font-bold uppercase text-gray-800"
+    >
+      Quick setup
+    </h3>
+    <div class="m-3">
+      <div class="mt-2 flex justify-between">
+        <h4 class="text-lg py-2">Update config</h4>
+        <div>
+          <button (click)="updateHelpshiftConfig()">
+            Update Helpshift Config
+          </button>
+        </div>
+      </div>
+      <div class="mt-2 flex justify-between">
+        <h4 class="text-lg py-2">Login user</h4>
+        <div>
+          @if(isLoggedIn){
+          <button (click)="onLogout()">Logout</button>
+          }@else{
+          <button (click)="onLogin()">Login</button>
+          }
+        </div>
+      </div>
+      <div class="mt-2 flex justify-between">
+        <h4 class="text-lg py-2">Full Privacy</h4>
+        <div class="flex flex-col space-y-2 p-2">
+          <form>
+            <div class="inline-flex items-center mr-4">
+              <input
```

Comment:
[P3] - Can we use [formControl](https://angular.dev/guide/forms/reactive-forms) here for making it reactive?